source/cli.rst

```diff
@@ -58,10 +98,22 @@ Description
 
 Define cache configuration
 
+Arguments/Options
+****************
+
+There are no arguments for this command
+
+Options:
+
+- ``--context`` (required) - Cache context (i.e. 'core' or 'plugin:plugin_name') (default = core)
+- ``--dsn`` (required) - Cache system DSN (no default)
+- ``--use-default`` - Unset cache configuration to use default filesystem cache for given context (no default)
+- ``--skip-connection-checks`` - Skip connection checks (no default)
```

Comment:
`InputOption::VALUE_NONE` means that the option cannot have a value. It corresponds to a boolean flag.